Registration/RegUser.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+namespace Registration
+{
+    public class RegUser
+    {
+        public bool PasswordVerification (string password, string passwordCheck)
+        {
+            if (password == passwordCheck)
+            {
+                return true;
+            }
+            return false;

```

Comment:
Предлагаю немного упростить функцию.
```suggestion
           return password == passwordCheck;
```

Reply:
Предложенный вариант более для меня понятен.

Reply:
Принимаю ваши возражения